common/test/Configuration.Oms.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public static partial class Configuration
+    {
+        public static class Oms
```

Comment:
Oms? What is that?

Reply:
Refactored the class to a more familiar name: Log Analytics
Log Analytics is where ASC for IoT stores its data.
The test sends a security message and then queries that message in the Log Analytics workspace